src/content/community/grants/merge-data-challenge/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Merge data challenge
+description: Instructions, deadlines, and prize information for the Merge data challenge
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+# The Merge data challenge {#the-merge-data-challenge}
+
+_Up until four weeks after the Merge, the Ethereum Foundation is sponsoring a Merge data analysis and data visualization blog post challenge. Here are all the details you need._
+
+<Divider />
+
+![Two people holding hands in field with ethereum logo](./merge-data-challenge.jpg)
```

Comment:
Just noting this feels like an unusual place to have an image. But, it's probably the best solution as we don't have a template to support this (yet).